### ISSUE TITLE
Add downlinkMax and effectiveType Dimensions

### ISSUE THREAD

**Issue by nicjansma** [rum-archive/rum-archive#23]
https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/downlinkMax
https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType

**Comment by bluesmoon** [comment]
do we want to add it as a dimension or as another histogram?  I feel adding it as a dimension will increase the risk of having to drop rows because of the risk of user identification.  Including it as a histogram provides the data but makes it harder to filter.

**Comment by nicjansma** [comment]
Yeah good point, I'd like to look into the cardinality of both before we add them as dimensions.  `effectiveType` should only be `4` but `downlinkMax` might have a lot.  We could smartly bucket that better.

Along with that, we can probably look into the cardinality of some of the existing dimensions and see if we can reduce some of them in a smart way, for example by limiting to the Top N.

https://rumarchive.com/docs/tables/#dimensions

One example might be `DEVICEMODEL` which sees over 2000+ devices on the exports, which definitely hurts the long-tail aggregation.  If we limited that to the top 100 or so models, maybe we'd get better aggregation.

I can do some experimentation.
